inst/extdata/variable_details.csv

```diff
@@ -1620,4 +1620,6 @@ ADL_score_5,ADL_score_5_cat6_3,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007
 ADL_score_5,ADL_score_5_cat6_4,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",N/A,4,6,Needs help with at least 4 tasks,Needs help with at least 4 tasks,N/A,N/A,N/A,N/A,N/A,
 ADL_score_5,ADL_score_5_cat6_5,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",N/A,5,6,Needs help with at least 5 tasks,Needs help with at least 5 tasks,N/A,N/A,N/A,N/A,N/A,
 ADL_score_5,ADL_score_5_cat6_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",N/A,NA::a,6,not applicable,not applicable,N/A,N/A,N/A,N/A,N/A,
-ADL_score_5,ADL_score_5_cat6_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",N/A,NA::b,6,missing,missing,N/A,N/A,N/A,N/A,N/A,
\ No newline at end of file
+ADL_score_5,ADL_score_5_cat6_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",N/A,NA::b,6,missing,missing,N/A,N/A,N/A,N/A,N/A,
+time_quit_der,N/A,cont,"cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[SMK_09A_B, SMKG09C]",N/A,Func::time_quit_fun,N/A,N/A,N/A,Years,N/A,N/A,Time since quit,Time since quit smoking,Variable derived from various harmonized smoking variables
+time_quit_der,N/A,cont,"cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[SMK_09A_B, SMKG09C]",N/A,NA::b,N/A,N/A,N/A,Years,N/A,N/A,Time since quit,Time since quit smoking,
```

Comment:
Could you elaborate why we need this extra row?

Reply:
The extra rows are for the NA labels. I've pushed a commit that more accurately reflects these labels for NA::a & NA::b.